### DISCUSSION TITLE
Winning probability

### DISCUSSION THREAD

**Discussion by chianew** [Chia-Network/chia-blockchain#4555] [Support]
new plot to pool will be out at 17th of this month

Is there any winning probability difference between old plots and new plots?

Thank you

**Comment by lvcivs** [comment]
A plot is a plot and will have the same chance of winning. But how much you get out of it will depend. For solo farming you will get the full amount, while for pool farming  the reward will go to the pool and you will be regularly payed out a fraction of the pool's winnings (minus pool fees). On the other hand, if you have a low amount of plots and go solo you might not win for years and it could be preferable to get at least small fractional payouts from a pool.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
